GCRCatalogs/alphaq.py

```diff
@@ -110,21 +110,30 @@ def _collect_native_quantities(name, obj):
             'sersic_disk':              'morphology/diskSersicIndex',
             'sersic_bulge':             'morphology/spheroidSersicIndex',
             'ellipticity_true':         'morphology/totalEllipticity',
-            'ellipticity_disk_true':    'morphology/diskEllipticity',
-            'ellipticity_bulge_true':   'morphology/spheroidEllipticity',
             'ellipticity_1_true':       'morphology/totalEllipticity1',
             'ellipticity_2_true':       'morphology/totalEllipticity2',
-            'ellipticity_1_disk_true':  'morphology/diskEllipticity1',
-            'ellipticity_2_disk_true':  'morphology/diskEllipticity2',
-            'ellipticity_1_bulge_true': 'morphology/spheroidEllipticity1',
-            'ellipticity_2_bulge_true': 'morphology/spheroidEllipticity2',
             'size_true': (
                 lambda size1, size2, lum1, lum2: ((size1*lum1)+(size2*lum2))/(lum1+lum2),
                 'morphology/diskMajorAxisArcsec',
-                'morphology/spheroidMinorAxisArcsec',
+                'morphology/spheroidMajorAxisArcsec',
                 'LSST_filters/diskLuminositiesStellar:LSST_r:rest',
                 'LSST_filters/spheroidLuminositiesStellar:LSST_r:rest',
             ),
+            'Av' : (lambda lum_v, lum_v_dust: (-2.5*(np.log10(lum_v_dust/lum_v))),
```

Comment:
We use `A_v` and `R_v` in [our schema](https://github.com/LSSTDESC/gcr-catalogs/blob/master/GCRCatalogs/SCHEMA.md)